macros/stitch/base/stitch_fb_ad_insights.sql

```diff
@@ -8,26 +8,27 @@
 {% macro default__stitch_fb_ad_insights() %}
```

Comment:
is there any reason why we need multiple implementations of this model? it seems like they are basically identical...we should be able to merge these into a single implementation.

Reply:
ah, i completely overlooked this one! fixing it now